includes/class-system-info.php

```
@@ -0,0 +1,297 @@
+		public static function get_php(): array {
+			return array(
+				'version'             => phpversion() ? phpversion() : null,
+				'sapi'                => php_sapi_name() ? php_sapi_name() : null,
+				'memory_limit'        => ini_get( 'memory_limit' ) ? ini_get( 'memory_limit' ) : null,
+				'max_execution_time'  => ini_get( 'max_execution_time' ) ? ini_get( 'max_execution_time' ) : null,
+				'upload_max_filesize' => ini_get( 'upload_max_filesize' ) ? ini_get( 'upload_max_filesize' ) : null,
+				'post_max_size'       => ini_get( 'post_max_size' ) ? ini_get( 'post_max_size' ) : null,
+				'display_errors'      => ini_get( 'display_errors' ) ? ini_get( 'display_errors' ) : null,
+				'extensions_count'    => count( get_loaded_extensions() ),
+			);
+		}
```

PHPMD (2.15.0)

[error] The method get_php is not named in camelCase. (undefined)

(CamelCaseMethodName)